src/api/getContests.js

```diff
@@ -3,21 +3,18 @@
 import type { Contest } from '../types/contest';
 import judges from '../judges';
 
-function getContests(): Promise<Array<Contest>> {
-  let ans: Array<Contest> = [];
-  let promises: Array<Promise<Array<Contest>>> = [];
+async function getContests(): Promise<Array<Contest>> {
+  const ans: Array<Contest> = [];
+  const promises: Array<Promise<Array<Contest>>> = [];
   Object.keys(judges).forEach(judge => {
     promises.push(judges[judge].getContests().catch(
```

Comment:
This `judges[judge]` is bugging me. But I lack the experience to find a better way. Did you search for alternatives?